source/features/command-palette-navigation-shortcuts.tsx

```diff
@@ -0,0 +1,34 @@
+import delegate from 'delegate-it';
+import onetime from 'onetime';
+
+import features from '.';
+
+function commandPaletteKeydown(event: delegate.Event<KeyboardEvent>): void {
+	const {key, ctrlKey, target} = event;
+
+	if (!target || !ctrlKey || (key !== 'n' && key !== 'p')) {
+		return;
+	}
+
+	event.preventDefault();
+
+	const targetKey = key === 'n' ? 'ArrowDown' : 'ArrowUp';
+	target.dispatchEvent(new KeyboardEvent('keydown', {bubbles: true, key: targetKey, code: targetKey}));
+}
+
+function init(): void {
+	delegate(document, 'command-palette', 'keydown', commandPaletteKeydown);
+}
+
+const onMacOs = !!navigator.userAgent.match(/Macintosh/);
+
+if (onMacOs) {
+	void features.add(import.meta.url, {
```

Comment:
We should use this instead:
```suggestion
	void features.add(import.meta.url, {
		include: [
			isMac
		],
```